doc/onboarding.md

```diff
@@ -169,6 +169,7 @@ Landing a PR
     * `Reviewed-By: human <email>`
       * Easiest to use `git log` then do a search
       * (`/Name` + `enter` (+ `n` as much as you need to) in vim)
+      * Only include collaborators who have commented "LGTM"
```

Comment:
> Oh good point! I hadn't considered that. It might be worthwhile clarifying
that in the readme, I think :)

@jasnell submitted a pull-request https://github.com/nodejs/node/pull/7996 for this.